config.local.yaml

```diff
@@ -71,3 +74,18 @@ metrics:
       - ['USDCBRL']
       - ['EUROCEUR']
       - ['XOFEUR']
+  - source: BreakerBox.getRateFeedTradingMode(address rateFeed)(uint8)
+    schedule: 0/10 * * * * *
+    type: gauge
+    chains: all
+    variants:
+      - ['CELOUSD']
+      - ['CELOEUR']
+      - ['CELOBRL']
+      - ['CELOXOF']
+      - ['USDCUSD']
+      - ['USDCEUR']
+      - ['USDCBRL']
+      - ['EUROCEUR']
+      - ['XOFEUR']
```

Comment:
i think this needs to bee EURXOF

Reply:
we have both directions

Reply:
hmm I think this refers to the variable name at line 7 and not the actual name of the rate feed.

I can change the name for consistency but that would also affect the label names in grafana, causing a weird gap in all metrics

![image](https://github.com/mento-protocol/aegis/assets/17413894/bc6f6a08-3669-4dc1-a4fc-d51e1e6bcc65)